Lib/idlelib/pyshell.py

```diff
@@ -8,6 +8,14 @@
     print("** IDLE can't import Tkinter.\n"
           "Your Python may not be configured for Tk. **", file=sys.__stderr__)
     raise SystemExit(1)
+
+if sys.platform == 'win32':
+    import ctypes
+    try:
+        ctypes.OleDLL('shcore').SetProcessDpiAwareness(1)
```

Comment:
Please define `PROCESS_SYSTEM_DPI_AWARE = 1` instead of using a magic number. It wouldn't hurt to quote MSDN in a brief comment that explains this value and also the MSDN URL for the [`PROCESS_DPI_AWARENESS`](https://msdn.microsoft.com/en-us/library/dn280512) enumeration.

Reply:
I will make a new PR for this.